src/utils/fetchClients.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'any' type. Try to use a more specific type or create a custom type for the data being sent to the server. For example, you can define an interface or type depending on the structure of the data 🌟